combat-trainer.lic

```diff
@@ -1577,7 +1583,12 @@ class SpellProcess
     return if game_state.casting
     return unless @training_spells
     return if @training_spells_max_threshold && game_state.npcs.length > @training_spells_max_threshold
-    return if DRStats.mana < @training_spell_mana_threshold
+    if DRStats.mana < @release_cyclic_threshold && @release_cyclic_on_low_mana == true
+      release_cyclics
+      return
+    elsif DRStats.mana < @training_spell_mana_threshold 
```

Comment:
Why change this to `elsif` instead of keeping the `return if...` guard clause? If I'm understanding this correctly the two conditionals aren't really related to each other (aside from both being "check for types of low mana"), and this seems to artificially couple them.

Reply:
True.  I think I ended up there while contemplating different tests.  This does work in both cases but I agree they're not necessarily mutually exclusive.